CorsixTH/Lua/world.lua

```diff
@@ -1143,7 +1148,7 @@ function World:onEndDay()
         -- The level uses random emergencies, so just create one.
         local_hospital:createEmergency()
       else
-        control = control[self.next_emergency_no]
+        control = self.next_emergency
```

Comment:
Predates you, but control shouldn't be used for both the list of all emergencies and for a particular one. You could directly use self.next_emergency below and get rid of this line.